daemon/logger/copier_test.go

```diff
@@ -130,3 +132,36 @@ func TestCopierSlow(t *testing.T) {
 	case <-wait:
 	}
 }
+
+func BenchmarkScannerCopier(b *testing.B) {
```

Comment:
These benchmarks exhaust all the RAM memory and start using lots of swap or they crash when memory constraints are set for the container.

Could you check this out, please? The commands I've used were the same ones you've used. The system I was testing on has 8 GB of RAM and 4 cores.